extensions/wikia/PortableInfobox/services/Parser/XmlParser.php

```diff
@@ -51,8 +51,21 @@ public function getDataFromNodes( \SimpleXMLElement $xmlIterable ) {
 	 */
 	public function getDataFromXmlString( $xmlString ) {
 		wfProfileIn( __METHOD__ );
+		$global_libxml_setting = libxml_use_internal_errors();
+		libxml_use_internal_errors( true );
 		$xml = simplexml_load_string( $xmlString );
 		if ( $xml === false ) {
+			$errors = libxml_get_errors();
+			foreach ( $errors as $xmlerror ) {
+				if ( $xmlerror->level >= LIBXML_ERR_ERROR ) {
```

Comment:
I was thinking more along the lines of
->warning
->error
->critical
for each of LIBXML error levels